server/meca/file-generators/article.js

```diff
@@ -1,7 +1,8 @@
 const xmlbuilder = require('xmlbuilder')
 const lodash = require('lodash')
 const config = require('config')
-const { User } = require('@elifesciences/xpub-model')
+const elifeApi = require('@elifesciences/xpub-model/entities/user/helpers/elife-api')
```

Comment:
I suggest we expose elife-api at the top level of the package.

```suggestion
const { elifeApi } = require('@elifesciences/xpub-model')
```

Reply:
Even better would be to expose the function as a static method on the `Team` model.

```suggestion
const { Team } = require('@elifesciences/xpub-model')
```